src/frontend/csrs-portal/src/app/components/landing/landing.component.scss

```diff
@@ -0,0 +1,589 @@
+@import "palette";
+@import "bootstrap/bootstrap-imports";
+
+h1 {
+  font-size: 3em;
+  line-height: 1.1em;
+  margin-bottom: 50px;
+  font-weight: bold;
+  text-shadow: 2px 2px 4px #000;
+}
+
+h3 {
+  font-size: 1.875em;
+  line-height: 1.4em;
+  font-weight: bold;
+  margin-bottom: 10px;
+  color: theme-palette(blue);
+}
+
+h4 {
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-bottom: 20px;
+  color: theme-palette(blue);
+}
+
+.lead {
+  font-size: 1.5em;
+  margin-bottom: 10px;
+  line-height: 1.5em;
+  text-shadow: 2px 2px 4px #000;
```

Comment:
remove box shadow here also

Reply:
It was removed.

Reply:
Talked to Kartheek. As per my understanding he will tackle any remaining cleanup as part of a new cleanup and refactoring ticket he is working on.